### ISSUE TITLE
[Bug] in msidentity-app-sync.exe/Microsoft.Identity.Web.ProjectTemplates

### ISSUE THREAD

**Issue by tim-biasoft** [AzureAD/microsoft-identity-web#1148]
**Which version of Microsoft Identity Web are you using?**
msidentity-app-sync.exe 1.0.0
Microsoft.Identity.Web.ProjectTemplates 1.8.2 (also tried 1.9.1)

**Where is the issue?**
msidentity-app-sync.exe / webapp2 template

**Is this a new or an existing app?**
new

**Repro**

```poweshell
dotnet new webapi2 --auth SingleOrg
msidentity-app-sync.exe
```

**Expected behavior**
App scaffolded and configured

**Actual behavior**
Error encountered with sign-in. See error message for details:
AADSTS90002: Tenant 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx' not found. This may happen if there are no active subscriptions for the tenant. Check to make sure you have the correct tenant ID. Check with your subscription administrator.

**Possible solution**
Remove Domain from appsettings.json before running msidentity-app-sync.exe
```json
"Domain": "yourDomain", 
```

**Comment by jmprieur** [comment]
@tim-biasoft:
- you want to make sure you're signed-in in Visual Studio or Azure CLI with you home tenant account. You might need to re-enter your credentials in Visual Studio?
- otherwise, if you want to use a specific tenant, use --tenant-id

**Comment by tim-biasoft** [comment]
@jmprieur I was signed in with Visual Studio and Azure CLI. It worked straight away after I removed Domain from appsettings.json which kind of rules out any authentication issue. Running these two commands back to back:
```powershell
dotnet new webapi2 --auth SingleOrg
msidentity-app-sync.exe
```
Should work, right?

**Comment by jmprieur** [comment]
@tim-biasoft 
thanks very much for the heads-up. I've proposed a PR for this scenario.

**Comment by tim-biasoft** [comment]
@jmprieur Glad I could help in some way, I think msidentity-app-sync and the templates are a very important and useful feature 👍

**Comment by jmprieur** [comment]
Still need testing (automatically closed by GitHub)

**Comment by jennyf19** [comment]
Included in [1.9.2 release](https://github.com/AzureAD/microsoft-identity-web/releases/tag/1.9.2)
